docs/tutorials/cargo_tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+bin/
+.crates.toml
```

Comment:
what's this?

Reply:
They were files that cargo was making when it downloaded stuff from crates.io; they were empty and then they appeared to contain hashes of the crates or something. I didn't look too closely.